Reading List/Reading List/Models/BookController.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  BookController.swift
+//  Reading List
+//
+//  Created by Ian Becker on 5/19/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import Foundation
+
+class BookController {
+    var books: [Book] = []
+    var readBooks: [Book] {
```

Comment:
great job using the .filter higher order function inside of a computed property!!!!